balancer/weightedroundrobin/balancer.go

```diff
@@ -45,6 +47,43 @@ import (
 // Name is the name of the weighted round robin balancer.
 const Name = "weighted_round_robin"
 
+var (
+	rrFallbackHandle = estats.RegisterInt64Count(estats.MetricDescriptor{
+		Name:           "grpc.lb.wrr.rr_fallback",
+		Description:    "This metric is experimental. Number of scheduler updates in which there were not enough endpoints with valid weight, which caused the WRR policy to fall back to RR behavior.",
```

Comment:
FWIW I preferred Java's simpler, all-caps `EXPERIMENTAL. ___`

Reply:
Haha, ok switched to that.